app/components/users/library/library-entry.js

```diff
@@ -17,6 +17,20 @@ export default Component.extend({
     return get(this, 'media.unitCount') || '-';
   }).readOnly(),
 
+  /**
+   * Returns keys used in our translation file.
+   */
+  airingStatus: computed('media.{startDate,endDate,unitCount}', function() {
```

Comment:
We can move it to `models/media.js` with some minor changes.

```javascript
/**
 * Returns the translation key for the current airing status of this media.
 *
 * @returns {string} Translation Key
 */
airingStatus: computed('{startDate,endDate,unitCount}', function() {
  const start = get(this, 'startDate');
  const end = get(this, 'endDate');
  if (start && (start.isBefore() || start.isSame())) {
    const isOneDay = get(this, 'unitCount') === 1;
    const isPastDay = end && (end.isBefore() || end.isSame());
    return (isOneDay || isPastDay) ? 'finished' : 'current';
  }
  return 'nya';
}).readOnly()
```

I'm not sure `{..,..}` will work without a context, If it doesn't change it to:

`computed('startDate', 'endDate', 'unitCount', ...`